tests/e2e/k8s/k8s_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 The ClusterLink Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8s
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+// TestE2EK8S runs all k8s e2e tests.
+func TestE2EK8S(t *testing.T) {
+	suite.Run(t, &TestSuite{})
```

Comment:
If I want to run just a single test, how is possible?

Reply:
e.g.
```
 go test -v ./tests/e2e/k8s -count=1 -testify.m TestConnectivity
 ```

Reply:
Maybe it would help if we could write it in a comment/document.

Reply:
Agree, this should be documented in the doc explaining on testing.
Another thing that needs to be documented is that you can use `DEBUG=1 go test ...` to enable debug logging, and that the logs are saved to `/tmp/clusterlink-k8s-tests`.

Reply:
Can you open an issue about that, unless you are going to do it immediately?

Reply:
#153